tools/SetupFlow/DevHome.SetupFlow.Common/WindowsPackageManager/WindowsPackageManagerManualActivationFactory.cs

```diff
@@ -30,31 +30,31 @@ public WindowsPackageManagerManualActivationFactory()
     {
     }
 
-    protected override T CreateInstance<T>(Guid clsid, Guid iid)
+    protected override unsafe T CreateInstance<T>(Guid clsid, Guid iid)
     {
-        var pUnknown = IntPtr.Zero;
+        void* pUnknown = null;
+
         try
         {
-            var hr = WinGetServerManualActivation_CreateInstance(clsid, iid, 0, out var instance);
+            var hr = WinGetServerManualActivation_CreateInstance(in clsid, in iid, 0, out pUnknown);
             Marshal.ThrowExceptionForHR(hr);
-            pUnknown = Marshal.GetIUnknownForObject(instance);
-            return MarshalInterface<T>.FromAbi(pUnknown);
+            return MarshalInterface<T>.FromAbi((IntPtr)pUnknown);
         }
         finally
         {
             // CoCreateInstance and FromAbi both AddRef on the native object.
             // Release once to prevent memory leak.
-            if (pUnknown != IntPtr.Zero)
+            if (pUnknown is not null)
             {
-                Marshal.Release(pUnknown);
+                Marshal.Release((IntPtr)pUnknown);
             }
         }
     }
 
-    [DllImport("winrtact.dll", EntryPoint = "WinGetServerManualActivation_CreateInstance", ExactSpelling = true, PreserveSig = true)]
-    private static extern int WinGetServerManualActivation_CreateInstance(
-        [In, MarshalAs(UnmanagedType.LPStruct)] Guid clsid,
-        [In, MarshalAs(UnmanagedType.LPStruct)] Guid iid,
+    [DllImport("winrtact.dll", ExactSpelling = true)]

```

Comment:
Whether you agree with my suggested change above or not, this import is inconsistent.

Reply:
Resolving this, as after https://github.com/microsoft/devhome/pull/930/commits/31871e136d417ef2889cea1a3921a53515387305 it's now consistent with other imports.